doc/plugin_agent_svidstore_sconecas_sessionmanager.md

```diff
@@ -17,10 +17,11 @@ When the SVIDs are updated, the plugin takes care of creating (or updating) sess
 
 | Configuration                           | Description |
 | ----------------------------------------| ----------- |
-| cas_address                             |  CAS HTTPS connection string |
+| cas_connection_string                   |  CAS HTTPS connection string |
 | cas_client_certificate                  |  Client certificate used to authorize the plugin |
 | cas_client_key                          |  Client certificate key |
 | trust_anchor_certificate                |  CAS identity certiticate: the plugin uses this certificate to ensure connections only with a known, attested CAS |
+| insecure_skip_verify_tls                |  Skip TLS verification in connection with the configured CAS (**do not use in production**) |
```

Comment:
it would be good to put an example of using skip tls, to know the type of variable. when I tested it I judged it as true or false